test/integration/documentation_controller_branding_test.rb

```diff
@@ -7,9 +7,44 @@ def test_docs_redirect_branded
     assert_redirected_to "https://docs.redhat.com/documentation/en-us/red_hat_satellite/#{ForemanThemeSatellite.documentation_version}/html/managing_hosts/registering_hosts_to_server_managing-hosts#Registering_Hosts_by_Using_Global_Registration_managing-hosts"
   end
 
+  def test_docs_dynamically_generated
+    get "/links/docs/Managing_Content?chapter=Products_and_Repositories_content-management"
+
+    assert_redirected_to "https://docs.redhat.com/documentation/en-us/red_hat_satellite/#{ForemanThemeSatellite.documentation_version}/html/managing_content/importing_content_content-management#Products_and_Repositories_content-management"
+  end
+
+  def test_docs_with_mapped_guide
+    get "/links/docs/Managing_Configurations_Ansible?chapter=Importing_Ansible_Roles_and_Variables_ansible"
+
+    assert_redirected_to "https://docs.redhat.com/documentation/en-us/red_hat_satellite/#{ForemanThemeSatellite.documentation_version}/html/managing_configurations_using_ansible/getting_started_with_ansible_in_satellite_ansible_integration#Importing_Ansible_Roles_and_Variables_ansible"
+  end
+
+  def test_docs_chapter_does_not_exist
+    get "/links/docs/Managing_Content?chapter=does-not-exist"
+
+    assert_redirected_to "https://docs.redhat.com/documentation/en-us/red_hat_satellite/#{ForemanThemeSatellite.documentation_version}/html-single/managing_content/index#does-not-exist"
+  end
+
   def test_docs_redirect_unknown_chapter
     get "/links/docs/Managing_Hosts"
 
     assert_redirected_to "https://docs.redhat.com/documentation/en-us/red_hat_satellite/#{ForemanThemeSatellite.documentation_version}/html/managing_hosts/"
   end
+
+  def test_many
```

Comment:
I'm not sure if we want to keep this, but it was a simple way for me to test if the old `DOCS_GUIDES_LINKS` were properly converted